history/petr-troyanskii.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Early machine translation scholar
+---
+# Petr Troyanskii
+Petr Troyanskii was an educator and scholar. In 1935, he filed and received USSR Patent 40995 for his invention. 
+
+The machine selected and typed words when translating from one language to another 
+or across multiple languages simultaneously.
+
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto. In the same years, 
+[Georges Artsrouni] (georges-artsrouni.md) independently filed a patent in Paris. 
```

Comment:
No space, and use the full path:

[Georges Artsrouni](/people/georges-artsrouni.md)